config/environments/staging.rb

```diff
@@ -88,6 +88,4 @@
   config.register_phase = :test
   config.register_url = 'test.openregister.org/load-rsf'
   config.register_ssl = true
-  config.register_username = ENV['COUNTRY_REGISTER_USERNAME']
```

Comment:
these are no longer used now we are reading from the secrets store. We may use them as an override (but on localhost/development only)

Reply:
I think we should remove this config (`config.register_username`) from all environment files

Reply:
My thinking is that they may still be useful on development, e.g. use their existence as an override for local testing, where the credentials should not be in the secrets store.

Reply:
...I can delete them in this PR and add them back if I find a use for them, since they are unused currently.

Reply:
I don't think we'll use them. The secrets file is ignored from git so we can have a local copy of that for test/development